app/views/shared/_dap_analytics.html.erb

```diff
@@ -1,2 +1,8 @@
 <!-- <%= t('notices.dap_participation') %> -->
-<script async src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&subagency=TTS" id="_fed_an_ua_tag"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=<%= IdentityConfig.store.dap_measurement_id %>"></script>
+<%= javascript_tag(nonce: true) do %>
+window.dataLayer = window.dataLayer || [];
+function gtag() { dataLayer.push(arguments); }
+gtag('js', new Date());
+gtag('config', '<%= IdentityConfig.store.dap_measurement_id %>');
```

Comment:
Is it worth escaping with `j` just in case? https://api.rubyonrails.org/classes/ActionView/Helpers/JavaScriptHelper.html#method-i-escape_javascript

```suggestion
gtag('config', '<%= j(IdentityConfig.store.dap_measurement_id) %>');
```

Reply:
Yep, good idea.